br/pkg/lightning/backend/external/writer.go

```diff
@@ -392,8 +409,15 @@ func (w *Writer) flushKVs(ctx context.Context, fromClose bool) (err error) {
 			zap.Any("rate", float64(savedBytes)/1024.0/1024.0/time.Since(ts).Seconds()))
 	}()
 
-	slices.SortFunc(w.writeBatch[:], func(i, j common.KvPair) int {
-		return bytes.Compare(i.Key, j.Key)
+	sorty.MaxGor = max(8, uint64(variable.GetDDLReorgWorkerCounter()))
```

Comment:
maybe add a parameter to the builder to better reuse by other component

Reply:
We can unify all the parameters in the later PR.